Source/RxComponents/LMCCavityModes.hh

```diff
@@ -49,6 +49,7 @@ namespace locust
 
         private:
             std::vector<std::vector<std::vector<std::vector<double>>>> fVoltagePhase;
+            std::vector<double> fChannelPhaseOffsetDeg;
```

Comment:
This variable is labeled as "Deg" in the name but is actually in radians. This could be confusing, and I'd recommend dropping "Deg" from the name